src/test/SampleCustomLoader/Makefile

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2020 Intel Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+OUT_LIB = libsampleloader.so
+
+LIB_INC = -Iinclude -I../../ -Irapidjson-1.1.0/include
+CFLAGS = -fPIC -Wl,-rpath,. -fpic 
+LDFLAGS = -ldl -lpthread
+
+all:
```

Comment:
Build files for example library should not be under `src/test` directory. This directory contains files required for unit test execution. Maybe add separate directory for custom loader examples?

Reply:
Right now the SampleCustomLoader is being built using Makefile. Need to move the build to beezel. That will take care of this.Will do that along with unit tests